Contest_API.md

```diff
@@ -253,7 +253,7 @@ Properties for file reference objects:
 
 | Name     | Type      | Description
 | -------- | --------- | -----------
-| href     | string ?  | URL where the resource can be found. Relative URLs are relative to the `baseurl`. Must point to a file of intended mime-type. Resource must be accessible using the exact same (possibly none) authentication as the call that returned this data.
+| href     | string ?  | URL where the resource can be found. Relative URLs are relative to the `baseurl`. Must point to a file of intended mime-type. Resource must be accessible using the exact same (possibly none) authentication as the call that returned this data. Required when served from an API, but optional inside a [Contest Package].
```

Comment:
I don't think this should be added here. IMO it's more confusing.

Reply:
I've kept this for now as I think we *do* need to mention this somewhere and we don't have a separate document to specify the requirements for using the "bare specification" in the context of serving it as an API.

Reply:
I still don't like this. I suggest the following (inspired by @nickygerritsen)

- Make `href` non-optional
- Put the added sentence in parathesis with a "Note: ".

```suggestion
| href     | string    | URL where the resource can be found. Relative URLs are relative to the `baseurl`. Must point to a file of intended mime-type. Resource must be accessible using the exact same (possibly none) authentication as the call that returned this data. (Note: this is actuallu optional inside a [Contest Package]).
```